Cargo.toml

```diff
@@ -65,5 +65,5 @@ object = { opt-level = 3 }
 rustc-demangle = { opt-level = 3 }
 
 [patch.crates-io]
-parquet2 = { version = "0.12", optional = true, git = "https://github.com/datafuse-extras/parquet2", rev = "03c108b" }
+parquet2 = { version = "0.12", optional = true, git = "https://github.com/datafuse-extras/parquet2", rev = "07176f1f" }
```

Comment:
This is a temporary fix?
https://github.com/datafuse-extras/parquet2/pull/3/commits/ea7820dc4f96bce29bae4d2def57b8ba67fd2f50